app/templates/components/forms/form-input.hbs

```diff
@@ -0,0 +1,16 @@
+<input 
+  data-test-input-field
+  disabled={{@disabled}}
+  name={{@name}}
+  value={{@value}}
+  placeholder={{@placeholder}}
+  autocomplete={{@autocomplete}}
+  autofocus={{@autofocus}}
+  type="text"
+  class="{{concat "travis-form__field-input travis-form__field-component " @class}}"
```

Comment:
I think with the use of `...attributes` below, most or all of these don't need to be made into parameters, as any html attributes you put on the component will be forwarded to this element. The one I'm a little unsure about though is `class`, since we have some existing classes that we definitely want on the element. I _think_ that splattributes is smart about combining attribute values like this, but I haven't tested it.